lib/eyaml/cli.rb

```diff
@@ -0,0 +1,74 @@
+class EYAML
+  class CLI < Thor
+    class_option :keydir, aliases: "-k", type: :string, default: EYAML::DEFAULT_KEYDIR, desc: "Directory containing EYAML keys"
+
+    method_option :json, type: :boolean, aliases: "-j", desc: "output will be formatted as JSON", default: false
+    desc "encrypt", "(Re-)encrypt one or more EYAML files"
+    def encrypt(*files)
+      files.each do |file|
+        file_path = Pathname.new(file)
+        next unless file_path.exist?
+
+        bytes_written = EYAML.encrypt_file_in_place(
+          file_path,
+          keydir: options.fetch(:keydir),
+          output_format: (options.fetch(:json) ? :json : :yaml)
```

Comment:
This should default to the current file format e.g. secrets.ejson -> json, secrets.eyaml -> yaml